CefSharp.BrowserSubprocess.Core/CefBrowserWrapper.h

```diff
@@ -35,6 +43,8 @@ namespace CefSharp
             CefSharp::Internals::JavascriptCallbackRegistry^ get();
         }
 
+        void SendProcessMessage(CefProcessId target_process, CefRefPtr<CefProcessMessage> message);
```

Comment:
Can likely remove the `target_process` param and just hard code it as it's pretty much guaranteed to always be sending messages to the `Browser` process. If you feel strongly enough to keep it then `target_process` should be renamed to `targetProcess`.